OpenRVS/classes/OpenHTTPClient.uc

```diff
@@ -155,18 +156,18 @@ function StartRequest(string url, string callbackName)
 	local ParsedURL parsed;
 	local HttpResponse resp;
 
-	self.kiloBuffer.Remove(0, self.kiloBuffer.Length);
+	kiloBuffer.Remove(0, kiloBuffer.Length);
 
 	if (callbackName != "")
-		self.CallbackName = callbackName;
+		CallbackName = callbackName;
```

Comment:
Unreal is not case sensitive with variable/class names so the string passed to the function should get a different name.  Now that I think of it, this is probably one of the cases where it makes sense to refer to self.VariableName.  But it makes more sense to give the argument a better name.  EG
function StartRequest(string url, string callbackDesired)
and
CallbackName = callbackDesired;
or something like that.